src/Tools/Directory.Build.props

```diff
@@ -21,4 +22,8 @@
     <NativeArtifactDirectories Include="$(ArtifactsBinDir)Windows_NT.x64.$(Configuration)" TargetRid="win-x64" LibraryPrefix="" LibraryExtension=".dll" SymbolsExtension=".pdb" />
     <NativeArtifactDirectories Include="$(ArtifactsBinDir)Windows_NT.x86.$(Configuration)" TargetRid="win-x86" LibraryPrefix="" LibraryExtension=".dll" SymbolsExtension=".pdb" />
   </ItemGroup>
+
+  <ItemGroup Condition=" '$(IsPackable)' == 'true' and '$(ThirdPartyNotice)' != ''">
+    <None Include="$(ThirdPartyNotice)" Pack="true" PackagePath="notices" Visible="false" />
```

Comment:
Can we call it `ThirdPartyNoticeFilePath` to make it clear this is a file path rather than the contents of the notice?